### ISSUE TITLE
Username Invalid token for Singularity CE endpoint

### ISSUE THREAD

**Issue by pfunzowalter** [singularityhub/sregistry#442]
**Describe the bug**
Hi, I am currently having trouble adding an endpoint to authenticate and then push containers for our registry. The issue arises due to an access token being invalid. I am using the [mozilla-django-oidc](https://mozilla-django-oidc.readthedocs.io/en/stable/installation.html) as a login plugin to the registry.  I have tried to add the token manually to the remote.yaml file as suggested [here](https://singularityhub.github.io/sregistry/docs/client) but it does not work. The bug looks like below.

**The Client logs**

`singularity --debug remote status
DEBUG   [U=10368,P=2900230]persistentPreRun()            Singularity version: 3.11.3
DEBUG   [U=10368,P=2900230]persistentPreRun()            Parsing configuration file /usr/local/etc/singularity/singularity.conf
DEBUG   [U=10368,P=2900230]handleConfDir()               /users/walter/.singularity already exists. Not creating.
DEBUG   [U=10368,P=2900230]handleRemoteConf()            Ensuring file permission of 0600 on /users/walter/.singularity/remote.yaml
INFO    [U=10368,P=2900230]RemoteStatus()                Checking status of default remote.
SERVICE    STATUS  VERSION  URI
Keyserver  OK      v1.0.0   https://sregistry.idia.ac.za
Library    OK      v1.0.0   https://sregistry.idia.ac.za
Token      OK      v1.0.0   https://sregistry.idia.ac.za

Authentication token is invalid (please login again).
FATAL   [U=10368,P=2900230]func61()                      error response from server: Unknown`
  
**Here are the Application logs**
`sregistry-uwsgi-1  | Unauthorized: /v1/token-status
sregistry-uwsgi-1  | [pid: 29|app: 0|req: 8/20] 154.114.10.245 () {34 vars in 486 bytes} [Sun Dec  3 05:39:43 2023] GET /v1/token-status => generated 38 bytes in 493 msecs (HTTP/1.1 401) 8 headers in 273 bytes (1 switches on core 1)
sregistry-uwsgi-1  | [pid: 28|app: 0|req: 3/21] 165.73.97.100 () {58 vars in 1151 bytes} [Sun Dec  3 05:40:07 2023] GET / => generated 8739 bytes in 352 msecs (HTTP/1.1 200) 7 headers in 222 bytes (1 switches on core 3)`

**To Reproduce**
It is documented [here](https://singularityhub.github.io/sregistry/docs/client).
**Expected behavior**
Documented [here](https://singularityhub.github.io/sregistry/docs/client).

If applicable, add versions and screenshots to help explain your problem.

**Comment by vsoch** [comment]
I'm not familiar with that OIDC plugin or what it would mean to interact with the registry here, perhaps ask the folks that maintain it?

**Comment by pfunzowalter** [comment]
Yes, I logged in via the Web interface in the app. just that in this case using OIDC instead of SAML or the other four methods that come with the code. After a successful login using OIDC to the app in the web interface, I need to retrieve the Secrets (eg TOKEN=5027225717bf0030465db1e7099e496022c42181) by clicking the username-->token, which I eventually use when I need to push images to the registry. So to add the token for our registry endpoint I use the command line as shown above. My issue is that it looks like the token I am retrieving, in this case, is invalid when adding it to Singularity Client.
